Automattic/jetpack
# Remove use of subscribe as site subscriber

Fixes https://github.com/Automattic/gold/issues/190

## Proposed changes:

* Do not rely on subscribe_as_site_subscriber for tiers

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* See WPCOM PR
* Unit tests
```
➜ cd projects/plugins/jetpack                                                     
➜  jetpack git:(add/remove-user-of-subscribe_as_site_subscriber) pnpm test-extensions extensions/store/membership-products/test/selectors-test.js
```

```
 ➜ jetpack docker phpunit-multisite --php 7.0  --  --testsuite=subscriptions 
``
